### ISSUE TITLE
Allow totally clean font list

### ISSUE THREAD

**Issue by dakom** [jewish-interactive/ji-wysiwyg-editor#3]
e.g. to start from scratch so that we can supply _only_ the fonts we have pre-loaded as web fonts

**Comment by jpuri** [comment]
@dakom : is not possible to know what all fonts has been loaded on the page, is it possible that you pass this list to the editor component ?

**Comment by dakom** [comment]
Yes - that's the intent of this issue, to allow the containing app/component to manually supply the list.

It is up to the containing app/component to preload them before starting the editor :)

**Comment by jpuri** [comment]
I have already added Hebrew fonts: https://github.com/jewish-interactive/ji-wysiwyg-editor/issues/6. I will make this change also 👍.

**Comment by jpuri** [comment]
Font family prop has now been added to the editor and fonts are required to be passed.
```
        <JiEditor
          defaultValue={"<p>testing</p>"}
          onChange={this.updateEditorContent}
          fontFamily={[
            "Assistant",
            "Varela Round",
            "David Libre",
            "Frank Ruhl Libre"
          ]}
        />
```

**Comment by jpuri** [comment]
Released `ji-wysiwyg-editor@0.1.9`
